externs/shaka/player.js

```diff
@@ -954,6 +954,38 @@ shaka.extern.StreamingConfiguration;
 shaka.extern.AbrConfiguration;
 
 
+/**
+ * @typedef {{
+ *   enabled: boolean,
+ *   useHeaders: boolean,
+ *   sessionId: string,
+ *   contentId: string
+ * }}
+ *
+ * @description
+ *   Common Media Client Data (CMCD) configuration.
+ *
+ * @property {boolean} enabled
+ *   If <code>true</code>, enable CMCD data to be sent with media requests.
+ *   Defaults to <code>false</code>.
+ * @property {boolean} useHeaders
+ *   If <code>true</code>, send CMCD data using the header transmission mode
+ *   instead of query args.  Defaults to <code>true</code>.
```

Comment:
Query arg is default right, so useHeaders would default to false?

Reply:
good catch